### ISSUE TITLE
Update configuration guide to use Azure AD Conditional Access with CF Access apps

### ISSUE THREAD

**Issue by etec-masterofsynapse** [cloudflare/cloudflare-docs#7239]
### Which Cloudflare product does this pertain to?

Zero Trust

### Existing documentation URL(s)

https://developers.cloudflare.com/cloudflare-one/tutorials/azuread-conditional-access/#1-set-up-an-identity-provider-for-your-application

### Section that requires update

Set up an identity provider for your application

### What needs to change?

At the moment, this guide mentions that you should create a new IdP entry in CF One for every app that should be protected by a certain Conditional Access Policy, meaning if you would use 100 Conditional Access Policies, you need to create 100 IdP entries, each with their own secret and attached configs required (which also have to be maintained since secrets expire).

### How should it change?

I would reword that paragraph, suggesting to first design a normal IdP integration which could be used as a baseline to configure it with a baseline Conditional Access policy, requiring MFA and a modern auth client.
Then, if an app needs special protection, e.g. only being allowed to access it from a certain IP, then one would configure a second IdP for that CA policy.

Also, I feel that some functionality is redundant here, both AAD CA and CF One apps can limit the access to certain IP ranges, users/groups or countries. Which functionality provided by AAD CA policies (apart from requiring MFA or modern auth clients) was missing from CF One rules that an integration with AAD CA made sense?
Maybe this info, which specific functionality is only available when pairing AAD CA with CF One should also be put into the docs so we can agree on one best practice where to put which Access Rules.

**Comment by ranbel** [comment]
Thank you for the suggestions! I attempted to reword the IdP instance note in #7258.

> Maybe this info, which specific functionality is only available when pairing AAD CA with CF One should also be put into the docs so we can agree on one best practice where to put which Access Rules.

This idea is great in theory, but it would be challenging to keep the doc in sync with the latest AAD CA features. I'll have to discuss with my team.

**Comment by etec-masterofsynapse** [comment]
@ranbel I can see the challenge you are describing of having to keep the docs in sync with the AAD CA features, but maybe the following would be a remedy to that:
Currently, there are some definite duplicate features present when pairing AAD CA with CFOAP. These can be listed anyway in the docs since they are so basic that MSFT won't remove them anytime soon from AAD CA.
And when you look at the CFOne roadmap and don't see any soon-to-be-released "duplicate" features with AAD CA, you can just direct the admins to AAD CA, stating that they should by default use that, and CFOne on top, and if CFOne is standalone, obviously use all the CFOne features.

**Comment by ranbel** [comment]
#7258
